src/benchmarks/micro/MicroBenchmarks.csproj

```diff
@@ -118,6 +118,9 @@
     <None Update="libraries\System.IO.Compression\TestData\**">
       <CopyToOutputDirectory>PreserveNewest</CopyToOutputDirectory>
     </None>
+    <None Update="libraries\System.IO.Compression\TestZipFile.zip">
```

Comment:
I was thinking about suggesting to just zip `TestData` during `[GlobalSetup]` but then I've realized that if somebody adds new file to `TestData`, the time reported by the benchmark is going to report a regression.

So please keep the zip file. 👍